bookkeeper-server/src/main/java/org/apache/bookkeeper/processor/RequestProcessor.java

```diff
@@ -47,4 +49,11 @@ public interface RequestProcessor extends AutoCloseable {
      * Flush any pending response staged on all the client connections.
      */
     void flushPendingResponses();
+
+    /**
+     * Process a list of ParsedAddRequests.
+     * @param r
+     * @param channel
+     */
+    void processAddRequest(List<BookieProtocol.ParsedAddRequest> r, BookieRequestHandler channel);
```

Comment:
Looks like `void processRequest(Object r, BookieRequestHandler channel);` is the same as the new one. Because the argument is an object, we can reuse that method